docs/index.md

```diff
@@ -2,6 +2,35 @@
 
 ## About
 
-M3 is a metrics platform that is built on M3DB, a distributed timeseries database. The project monorepo can be found [here][m3].
+[M3](https://github.com/m3db/m3) is an open source metrics platform that was developed at Uber out of necessity.
+After using what was available as open source and finding we were unable to use them at our scale due to issues
+with their reliability, cost and operationally intensive nature, we built our own metrics platform piece by piece.
+We used our experience to help us build a native distributed time series database, a highly dynamic and performant
+aggregation service, query engine and other supporting infrastructure.
 
-[m3]: https://github.com/m3db/m3
+## Key Features
+
+Below are some of the key features of M3 that make it a great platform for time series data.
+
+* Distributed time series database (M3DB) that provides scalable storage and a reverse index of time series.
+* Side-car process (M3Coordinator) that allows M3DB to act as the long-term storage for Prometheus.
+* Distributed query engine (M3Query) with native support for PromQL and Graphite.
```

Comment:
> A distributed query engine, M3Query, [...]

same as above

Reply:
also, there's no world in which we don't eventually add M3QL - let's find a good way to call that out, even if it's something like

> [...] PromQL and Graphite (M3QL support forthcoming).